hack/mac-install-deps.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+# scrapes Dockerfile for wget commands for fetching current versions of tanzu carvel tools,
+#  replaces "linux" with "darwin" so we pull the osx images, cleans up lingering Docker and multiline-grep formatting,
```

Comment:
how is this meant to deal with checksums?

Reply:
I guess the script could have the checksums hardcoded and set as vars that could be `sed` into the script in place of where the linux checksums are.

Reply:
Optimistically?  :)  right now it removes them in one of the `grep -v` calls,  since the Dockerfile checksums were for the linux build. 

I felt ok being optimistic about checksums for a development or semi-manual environment setup, and we do still run the `<tool> version` command as a sanity check so there would be prompt feedback if the installed binary were inoperable. 

 Is there a different location from which we could try to scrape the correct checksums (and would that add enough complication to this little script that it became harder to use/read/maintain)?